rundeckapp/grails-app/services/rundeck/services/ReportService.groovy

```diff
@@ -424,7 +426,18 @@ class ReportService  {
             query.statFilter='cancel'
         }
     }
-    def getExecutionReports(ExecQuery query, boolean isJobs) {
+    def getExecutionReports(RdExecQuery query, boolean isJobs) {
+        ReportFilter reportFilter = new ReportFilter(
+                stat: 'status',
+                reportId: 'reportId',
+                jobId: 'jcJobId',
+                proj: 'ctxProject',
+                user: 'author',
+                node: 'node',
+                message: 'message',
+                job: 'reportId',
+                title: 'title',
+                tags: 'tags')
```

Comment:
use of ReportFilter here is not correct. This is a domain class and doesn't make sense to use it to pass as a parameter for the query field mapping. Aside from that, the purpose of the `filters` map used below is to map query parameters (i.e. URL parameters and form fields from the GUI query interface), to correct fields to query in the DB table.  E.g. the input form field "statFilter" is mapped to the domain class field "status".  Instead, the mapping from input parameter to query field, will probably make the most sense within the Gorm data provider. Passing the ReportFilter object should be removed, and the Map construction for the input to field mapping should be moved to the Gorm data provider.